homematic/data/run.sh

```diff
@@ -1,15 +1,13 @@
-#!/bin/bash
+#!/usr/bin/env bashio
 # shellcheck disable=SC1091
 set -e
 
-CONFIG_PATH=/data/options.json
-
-RF_ENABLE=$(jq --raw-output '.rf_enable' $CONFIG_PATH)
-RF_DEVICES=$(jq --raw-output '.rf | length' $CONFIG_PATH)
-WIRED_ENABLE=$(jq --raw-output '.wired_enable' $CONFIG_PATH)
-WIRED_DEVICES=$(jq --raw-output '.wired | length' $CONFIG_PATH)
-HMIP_ENABLE=$(jq --raw-output '.hmip_enable' $CONFIG_PATH)
-HMIP_DEVICES=$(jq --raw-output '.hmip | length' $CONFIG_PATH)
+RF_ENABLE=$(bashio::config 'rf_enable')
```

Comment:
None of these 6 variables are needed anymore and can be replaced in line with Bashio.
See the other comments I made on this PR.